rotkehlchen/exchanges/kraken.py

```diff
@@ -225,6 +239,19 @@ def history_event_from_kraken(
                 f'Failed to read ledger event from kraken {raw_event} due to {msg}',
             )
             return [], False
+
+    for event_set in receive_spend_events.values():
+        if len(event_set) == 2:
+            for event in event_set:
+                if event.event_type == HistoryEventType.RECEIVE:
+                    event.event_subtype = HistoryEventSubType.RECEIVE
+                else:
+                    event.event_subtype = HistoryEventSubType.SPEND
+                    event.balance.amount = AssetAmount(-event.balance.amount)
```

Comment:
Why to use `AssetAmount`? Shouldn't this be just an FVal?

Reply:
I got it from another line but seems that is not needed